redis/acknowledge.lua

```diff
@@ -8,7 +8,7 @@ local error = ARGV[2]
 local ttl = ARGV[3]
 redis.call('zrem', zset_key, test)
 redis.call('hdel', owners_key, test)  -- Doesn't matter if it was reclaimed by another workers
-local acknowledged = redis.call('sadd', processed_key, test)
+local acknowledged = redis.call('sadd', processed_key, test) == 1
```

Comment:
I think the mistake was to think Lua consider `0` as falsey.

Reply:
That's right 🤦